data/processed/data_config.csv

```diff
@@ -0,0 +1,32 @@
+column,type,max,min
+diagnosis,str,,
```

Comment:
Should we include column labels in this config, or in another. Seems strange to only have numerical data in the config file.